package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-plugin-lit": "^1.5.1",
     "husky": "^8.0.0",
     "lodash": "^4.17.21",
+    "markdown-magic": "^2.6.1",
```

Comment:
Including this and the other updates to the build scripts should not be included in this commit. Adding markdown-magic has nothing to do with creating a rounded button. 

Please update the body of the commit message to include that included with the actual feature also adds the initial support for using Markdown Magic.